src/components/multi-select/MultiSelect.tsx

```diff
@@ -54,6 +53,7 @@ const FilterSelector = <T extends Pick<CategoryNameInterface, '_id'>>({
   const [menuAnchor, setMenuAnchor] = useState<null | HTMLElement>(null)
   const [inputValue, setInputValue] = useState<string>('')
   const [selectedNames, setSelectedNames] = useState<string[]>([])
+  const AppButtonRef = useRef<HTMLButtonElement | null>(null)
```

Comment:
I think `appButtonRef ` should be better name, because we use CamelCase for ref variables

Reply:
` const handleInputChange = (e: ChangeEvent<HTMLInputElement>) => {
    setInputValue(e.target.value)
  }`
  
 Ref replaced by event.target